README.md

```diff
@@ -1,4 +1,36 @@
 # gRPC for dotnet
 
+## Experimental Only!
+
 This repository is currently experimental and work-in-progress.
-See https://github.com/grpc/grpc for the official version of gRPC C#.
+See https://github.com/grpc/grpc for the official version of gRPC C# (ready for production workloads).
+
+## The Plan
+
+We plan to implement a fully-managed version of gRPC for .NET that will be built on top of ASP.NET Core HTTP/2 server.
+Here are some key features:
+- API compatible with the existing gRPC C# implementation (your existing service implementations should work with minimal adjustments).
+- Fully interoperable with other gRPC implementations (in other languages and other platforms)
+- Good integration with the rest of ASP.NET Core ecosystem
+- High-performance (we plan to utilize some of the cutting edge performance features from ASP.NET Core and in .NET plaform itself)
+- We plan to provide a managed .NET Core client as well (possibly with limited feature set at first).
```

Comment:
nit: the bullet points inconsistently end with and without fullstops

Reply:
Done.